setup.py

```diff
@@ -34,7 +34,7 @@
         'sqlalchemy',
         'click',
         'datapackage',
-        'jtssql',
+        'jsontableschema',
```

Comment:
PR's code only need `jsontableschema-sql` package but it's not published on PyPi for now (we've just discussed it on FD slack channel=)